frontend/src/scenes/web-analytics/webAnalyticsLogic.tsx

```diff
@@ -254,27 +259,14 @@ export const webAnalyticsLogic = kea<webAnalyticsLogicType>([
             shouldStripQueryParams,
         }),
         setConversionGoal: (conversionGoal: WebAnalyticsConversionGoal | null) => ({ conversionGoal }),
-        setStateFromUrl: (state: {
-            filters: WebAnalyticsPropertyFilters
-            dateFrom: string | null
-            dateTo: string | null
-            interval: IntervalType | null
-            graphsTab: string | null
-            sourceTab: string | null
-            deviceTab: string | null
-            pathTab: string | null
-            geographyTab: string | null
-            isPathCleaningEnabled: boolean | null
-        }) => ({
-            state,
-        }),
         openModal: (tileId: TileId, tabId?: string) => {
             return { tileId, tabId }
         },
         closeModal: () => ({}),
         openAsNewInsight: (tileId: TileId, tabId?: string) => {
             return { tileId, tabId }
         },
+        setHasLoadedStateFromUrl: () => ({}),
```

Comment:
```suggestion
        setHasLoadedStateFromUrl: true,
```

Reply:
Hmm I tried this and it complained, might have just been that the types hadn't caught up yet, so I'll try this again

Reply:
Done